feedback/README.md

```diff
@@ -29,22 +56,36 @@ local.properties.
 If you have configured Recaptcha in your Sakai, Feedback will use it to
 validate unauthenticated technical feedback reports.
 
-Forcing the Tool into Sites
+
+Forcing the Tool onto Existing Sites
+---------------------------
+
+To backfill existing sites to have the Feedback Tool. you need to run a quartz
+job to add the tool to all the sites. There is a Job called "Backfill tool into sites"
+which when run can add a specific tool to all sites that match a partiular type.
+
+
+Forcing the Tool to Stay on Sites
 ---------------------------
 
-You can force all your sites to have a copy of this tool wit the following lines
+You can force all your sites to have a copy of this tool with the following lines
 in your sakai.properties
 
     # Make sure the feedback tool can't be removed from sites.
     poh.uneditables=sakai.feedback
     poh.unhideables=sakai.feedback
 
-then place a toolOrder.xml in your sakai home folder that marks the feedback tool
-as required. An example toolOrder.xml is in the root of the feedback project.
 
-To backfill existing sites to have the Feedback Tool. you need to run a quartz 
-job to add the tool to all the sites. There is a Job called "Backfill tool into sites"
-which when run can add a specific tool to all sites that match a partiular type.
+Turning Off Automatically Including the Tool in New Sites
+---------------------------------------------------------
+
+The way the Contact Us tool works by default is that, when you are creating a new
+ site, when you get to the 'Project Site Tools' page where you can choose the tools
+ to include on the new site, the Contact Us is automatically ticked and greyed out
+  (like the Site Info tool).  To turn off this feature aso that the tool is not ticked
+  or greyed out by default on this page, you need to remove the words "required=true"
+  in 2 places from the toolOrder.xml' file in the kernel module.
```

Comment:
What do you think of this README if I emailed it out to the sakai dev user list?